docsrc/providers.rst

```diff
@@ -118,6 +118,75 @@ Contentful
 
 Uses client :ref:`Contentful <client_contentful>`.
 
+Github
+======
+
+*Github* can `distribute` secrets to the `repository` scope only. It is also a config provider, and can be used as a 
+source for your secret and environment specs. In your SAM template, use `KEYDRA_CFG_PROVIDER=github` to 
+tell Keydra to look in Github for secrets to manage.
+
+This provider is really just an MVP to kickstart Github support, so other secret scopes have not yet
+been implemented.
+
+For a given secret specification, the `distribute` function will add an encrypted secret to the specified repository. See
+https://docs.github.com/en/actions/reference/encrypted-secrets.
+
+For example, for a secret spec of:
+
+.. code-block:: yaml
+
+   sample:
+      key: keydra_managed_sample
+      description: A example secret which exists in IAM
+      custodians: my_team
+      provider: IAM
+      rotate: nightly
+      distribute:
+      - config:
+            repository: keydraconfiguration
+            account_username: me
+         envs:
+            - dev
+         key: AWS_ACCESS_KEY_ID
+         provider: github
+         scope: repository
```

Comment:
Would it make sense to have this under `config` as well, like other provider-specific options?

Reply:
Moved it into config as suggested. This will be a breaking change for any existing bitbucket dists!